### ISSUE TITLE
The right upper menu disappear in narrow display

### ISSUE THREAD

**Issue by pongraczi** [laravel-enso/Enso#27]
Please check the following image. The upper-right menu disappears on narrow display.

![peek 2018-01-13 23-39](https://user-images.githubusercontent.com/5891983/34910699-0411256e-f8bb-11e7-844c-40d0021c667d.gif)

**Comment by aocneanu** [comment]
Yes I know, it was designed this way in the beginning and I totally forgot about that.

I will look into it.

**Comment by pongraczi** [comment]
Thanks, could you add a quick possibility to logout?
At this moment if I want to logout, I have to click to own profile.
I guess, it is possible to add this extra button (whatever) to the menubar, but maybe could be useful for others, too :)

**Comment by aocneanu** [comment]
I'm not sure that a quick logout will bring value to the project. We'll think about it.

Meanwhile I fixed the profile menu link for touch devices.

**Comment by pongraczi** [comment]
Thanks, no problem regarding the quick logout after the My profile visible in small displays.
